docs/modules/plugins/partials/proxy-properties.adoc

```diff
@@ -21,3 +21,8 @@
 |host.docker.internal
 |empty
 |Overrides the proxy host which will be passed to browser
+
+|proxy.publish-har-on-failure
+|true/false
+|false
+|Adds a har file for each failed step
```

Comment:
please add information on what will HAR contain in this case

Reply:
also please add the documentation regarding meta that can be used to disable HAR publishing